ara/views/host.py

```diff
@@ -21,6 +23,31 @@ def show_host(host):
     except models.NoResultFound:
         abort(404)
 
+    try:
+        facts = host.facts.one()
```

Comment:
Using a one-to-one relationship this becomes much simpler, because either `host.facts` will have a value or it will be None.  So you can skip this code here, and you don't need to pass `facts=facts` separately to the template, and the template can simply check if `host.facts` is has a value.